src/TerminalFighter/MainMenu/MainMenu.h

```diff
@@ -7,17 +7,17 @@
 
 #include "../TextArea/TextArea.h"
 
-enum Options { START, QUIT };
 
 class MainMenu {
 public:
     MainMenu(std::string font_path, int width, int height);
     ~MainMenu();
+    enum Options { START, QUIT };
```

Comment:
put this at the top of the public separated by a new line. This is a pretty important piece and should be easy to spot.